app/controllers/coronavirus_form/offer_other_support_controller.rb

```diff
@@ -32,6 +32,6 @@ def submit
   NEXT_PAGE = "business_details"
 
   def previous_path
-    offer_community_support_path
+    offer_care_qualifications_path
```

Comment:
```suggestion
    offer_care_path
```
This should be the offer care page, since we don't know whether they answered Yes/No to whether they can offer care, so may not have seen the qualification page.